log/corelog/console_logger.go

```diff
@@ -67,7 +67,7 @@ func (l consoleLogger) LogEvent(interface{}, EventLogFields) {
 
 }
 
-func addColor(level Level, message string) string {
+func AddColor(level Level, message string) string {
```

Comment:
Any idea why we're not using colorstring.go from utils?

Reply:
The idea was that in the long run only the high level logger should need and know about the colours. So it is not necessary to share it with others.